datadog-setup.php

```diff
@@ -326,7 +326,7 @@ function install($options)
                 if (is_truthy($options[OPT_ENABLE_APPSEC])) {
                     execute_or_exit(
                         'Impossible to update the INI settings file.',
-                        "sed -i 's@datadog.appsec.enabled \?=.*$\?@datadog.appsec.enabled = On@g' "
+                        "sed -i 's@;datadog.appsec.enabled \?=.*$\?@datadog.appsec.enabled = On@g' "
```

Comment:
```suggestion
                        "sed -i 's@;\? \?datadog.appsec.enabled \?=.*$\?@datadog.appsec.enabled = On@g' "
```
Maybe to handle upgrades?

Reply:
Done